### PR TITLE
CTCP-1293: IncidentCountry page added

### PR DESCRIPTION
**Pull request by BledarFeratiEY**

### CODE REVIEW

File: app/views/incident/IncidentCountryView.scala.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    @formHelper(action = routes.IncidentCountryController.onSubmit(mrn, mode), 'autoComplete -> "off") {
+
+        @errorSummary(form.errors)
+
+        @inputSelect(
+            field = form("value"),
+            selectType = OrdinarySelect(
+                heading = messages("incident.incidentCountry.heading"),
+                caption = Some(messages("incident.caption")),
```
Comment (jackheslop96) [inline app/views/incident/IncidentCountryView.scala.html:50]: Trailing comma

File: conf/messages.en [hunk 2]
```diff
@@ -276,7 +276,19 @@ locationOfGoods.addAdditionalIdentifier.heading = Do you want to add another ide
...
+# Incidents and Endorsements
+
+incident.caption = Incidents and Endorsements
+
 incident.incidentFlag.title = IncidentFlag
 incident.incidentFlag.heading = IncidentFlag
 incident.incidentFlag.checkYourAnswersLabel = IncidentFlag
 incident.incidentFlag.error.required = Select yes if IncidentFlag
+
+incident.incidentCountry.title = In which country is the place of incident in?
```
Comment (jackheslop96) [inline conf/messages.en:288]: I know this is only placeholder content but starting and ending the sentence with `in` isn't grammatically correct 😉

File: app/views/incident/IncidentCountryView.scala.html (hunk 1 above)
Comment (BledarFeratiEY) [reply app/views/incident/IncidentCountryView.scala.html:50]: Was an accident, fixed now.

**Review by ajchilts — APPROVED** [review]
LGTM after changes.

**Review by jackheslop96 — APPROVED** [review]

**Pull request merged**